CHANGELOG.md

```diff
@@ -0,0 +1,41 @@
+# Changelog
+
+Based on KeepAChangelog.
+Generated by **Documatic.**
+
+## Unreleased
```

Comment:
We already have released our first version, see https://github.com/gauravgahlot/argoflows/releases/tag/v0.1.0.

So, we should include services and their endpoints supported in this release. For that you can check https://github.com/gauravgahlot/argoflows/blob/847a0b53bd2bfc44321c01f24e80ff631f5772a6/README.md#supported-apis this table.